### DISCUSSION TITLE
Add revset function corresponding to `git sw` commits

### DISCUSSION THREAD

**Discussion by pokey** [arxanas/git-branchless#587] [Ideas]
Would be nice to have a revset function that returns the same set of commits that appear in the `git switch` quick pick. Was planning to use this for POC VSCode extension

Alternately, is there a simple way to express this set of commits using today's revset functionality?

**Comment by arxanas** [comment]
I think the commits are just `draft()`?

**Reply by pokey** [reply]
Didn't seem exactly the same.  It also includes the head of main I believe.  And for some reason it seemed to include a couple fewer commits, but I couldn't figure out which it was missing, as I wasn't sure how to dump the output to a file for diffing 🤔

**Reply by arxanas** [reply]
@pokey oh, you probably want the set of commits described here: https://github.com/arxanas/git-branchless/issues/492
